recline.make

```diff
@@ -5,7 +5,7 @@ api = 2
 libraries[recline][type] = libraries
 libraries[recline][download][type] = git
 libraries[recline][download][url] = "https://github.com/NuCivic/recline.js.git"
-libraries[recline][download][revision] = "072d186918afdd98206c16015b8a77d79dd4756d"
+libraries[recline][download][branch] = "error_508_compliance_civic_4733"
```

Comment:
Generally this looks good but we don't want to be pointing at a branch. Can you point this to the correct commit hash?